src/Tests/Microsoft.NET.TestFramework/Commands/TestCommand.cs

```diff
@@ -83,7 +84,23 @@ public ProcessStartInfo GetProcessStartInfo(params string[] args)
         public CommandResult Execute(params string[] args)
         {
             IEnumerable<string> enumerableArgs = args;
-            return Execute(enumerableArgs);
+            return ExecuteWithRetry(
```

Comment:
Can we pass the `Log` to this method so it can log the fact that it is waiting and retrying?

Reply:
done